Fix organisation of User Guide (#201)
AY2223S1-CS2103T-T11-2/tp: docs/UserGuide.md

## How to use this guide

First time using InTrack? We highly recommend reading the user guide in order. If you come across unfamiliar terms used in this user guide, their definitions may be found in the [glossary](#glossary).

Searching for information about specific features? You might find it helpful to navigate to the relevant sections via the [table of contents](#toc) for a more detailed explanations of individual features.
Alternatively, consider looking at the [command summary](#command-summary) for a brief outline of all the commands.

| Syntax                                                                                                                                   | Purpose                                                                                                                                                                     |
|------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `words in monospace font`                                                                                                                | Commands to be typed into InTrack                                                                                                                                           |

Reviewer: Is this always the case though?

### Deleting an internship application : `delete`

Deletes the internship application at the specified `INDEX` from InTrack.
`INDEX` refers to the index number of the desired entry as shown in the displayed internship list, and **must be a
positive unsigned integer**.

Format: `delete INDEX`

Example of usage:

* `delete 1`

Expected outcome:

* The first internship application in InTrack is deleted.

### Updating status of an internship application : `status`

Updates the status of the internship application at the specified `INDEX` from InTrack with 3 possible statuses
, `p` for "Progress", `r` for "Rejected" and `o` for "Offered".
`INDEX` refers to the index number shown in the displayed internship list and **must be a positive unsigned integer**.

Format: `status INDEX STATUS`, where `STATUS` must be either `p`, `o` or `r`

Example of usage:

* `status 1 o`

Expected outcome:

> * The status of the first internship application in InTrack will be updated to 'Offered' inside a small green box.

Reviewer: ```suggestion
* The status of the first internship application in InTrack will be updated to `Offered`.
```

Expected outcome:

* Edits the fields of the selected entry to match that in the input, such that the name of the company changes to
  Google, the position changes to Data Analyst, the email changes to google@gmail.com, the website changes to
  https://google.com, the tags are changed to just URGENT and the salary becomes $1000.
* Edits the position field of the selected entry to become SWE. All other details of the entry remain unchanged.

### Adding a task to a selected internship application : `addtask`

> Adds a task to a selected internship application at the selected `Internship` from InTrack.

Reviewer: ```suggestion
Adds a task to the currently selected internship application.

Format: `addtask TASK_NAME /at TASK_TIME`

* `TASK_NAME` refers to the name of the task to be added.
* `TASK_NAME` can take any value, but must not be left blank.
* `TASK_TIME` refers to the time the added task is due or at.
* `TASK_TIME` must be in the format `dd-MM-yyyy HH:mm`.

Example of usage:

* `addtask Technical Interview /at 12-01-2023 15:00`

Expected outcome:

* The selected `Internship` would be updated with the new task added in the list in an ascending manner, with the
  internship with the task that has the earliest date and time that is on or after the current time at the top.

### Deleting a task from a selected internship application : `deltask`

> Deletes the task at the specified index in the task list of a selected internship application.

Reviewer: ```suggestion
Deletes the task at the specified index in the task list of the currently selected internship application.
```
